blog-homepage/src/AuthorBio.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+  class AuthorBio extends React.Component {
+    static propTypes = {
+      author: PropTypes.object
+    }
+
+    render() {
+      const author = Object.values(this.props);
+      return (
+
+              <div>
+                <img src={author.map((info, i) => info.image)} alt='' />
```

Comment:
Since author is an object, you can just use `author.image` here. Same goes for the name